libmozdata/FileStats.py

```diff
@@ -18,7 +18,7 @@ class FileStats(object):
     """Stats about a file in the repo.
     """
 
-    def __init__(self, path, channel='nightly', node='tip', utc_ts=None):
+    def __init__(self, path, channel='nightly', node='tip', utc_ts=None, max_days=None):
```

Comment:
I think that originally I put max_days=3 as argument and Sylvestre asked me to put this value in a config file